### ISSUE TITLE
Shortcuts not working

### ISSUE THREAD

**Issue by HamzaM** [aleab/toastify#43]
Hello,

Using the latest version available on Windows and Spotify 1.0.75.483.g7ff4a0dc, the song is displayed on the bottom right as it is supposed to but the shortcuts I've assigned do not work and don't control the music.

Thanks for your work again, it is nice to see Toastify kept updated ;)

UPDATE: when pressing the button to map, remember that Ctrl or Shift might still be ticked above.

**Comment by aleab** [comment]
I'm reopening this issue and marking it as an improvement suggestion, since having to tick/untick key modifiers every time might be annoying; it's much better and more intuitive to check if they are really pressed when registering the hotkey.

**Comment by HamzaM** [comment]
Yes that's what I think would be more intuitive and logic as this is how it's supposed to be. If you could implement that, it would be easier for people and also avoid the issue I faced.
